api_app/analyzers_manager/observable_analyzers/criminalip_scan.py

```diff
@@ -0,0 +1,1340 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.analyzers_manager.exceptions import AnalyzerRunException
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+
+class CriminalIpScan(classes.ObservableAnalyzer):
+    url = "https://api.criminalip.io"
```

Comment:
can you create a base class that shares common attributes between the different analyzers? and a folder maybe to connect them? we did something similar already for other analyzers